### PR TITLE
Add testing to update atomic habit

### PR DESCRIPTION
**Pull request by BernardLesley**
Add JUnit testing for Update Atomic Habit

### CODE REVIEW

**Review by wenxin-c — COMMENTED** [review]
Overall good work, just some suggestions on coding style and naming conventions. Might need help from the rest for the naming convention comments as I'm not too sure as well. Other than that it looks good to me:D

File: src/main/java/seedu/duke/atomichabit/feature/AtomicHabit.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ public int getCount() {
         return count;
     }
 
-    public void setCount(int increment){
+    public void setCount(int increment) {
```
[the remark this answers is not in the document]
Comment (nichyjt) [reply src/main/java/seedu/duke/atomichabit/feature/AtomicHabit.java:27]: I agree with Wen Xin, `increaseCount` may be more descriptive.

File: src/test/java/seedu/duke/atomichabit/AtomicHabitTest.java [hunk 2]
```diff
@@ -41,6 +42,41 @@ public void addHabit_invalidCommand_exceptionThrown() {
...
+    @Test
+    public void updateHabit_indexOutOfBounds_exceptionThrown() throws AtomicHabitException {
+        // Test false command by user
+        addHabit_correct_output();
+        String habitIndex = "1000";
+        Command updateCommand = new UpdateCommand(habitIndex);
+        Assertions.assertThrows(AtomicHabitException.class, () -> {
+            updateCommand.execute(habitList);
+        });
+    }
```
Comment (nichyjt) [inline src/test/java/seedu/duke/atomichabit/AtomicHabitTest.java:79]: A nit, but for fuller coverage, test if negative values throw the appropriate error as well

File: src/main/java/seedu/duke/atomichabit/feature/AtomicHabit.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (YongbinWang) [reply src/main/java/seedu/duke/atomichabit/feature/AtomicHabit.java:27]: `increaseCount` will be more intuitive.

**Review by nichyjt — CHANGES_REQUESTED** [review]
Minor nits, please work on Wenxin's comments as well as mine and it LGTM!

**Comment by BernardLesley** [comment]
This fixes #67

**Review by nichyjt — APPROVED** [review]
Thanks for the fixes. LGTM!

**Pull request merged**